src/app/Main/Body/QueryPage/ExampleQueriesDialog/ExampleQueries.constants.ts

```diff
@@ -6,8 +6,10 @@ const defaultQuery2 = "?microorganism=Salmonella+spp.&samplingYear=2015";
 const defaultQuery3 =
     "?microorganism=CARBA-E.+coli&row=microorganism&column=samplingYear";
 
-export const defaultQueriesLists = {
+export type QueryCategories = "animal" | "food" | "animalFeed";
```

Comment:
Rename to QueryCategory, so it is one of the categories.

Reply:
Done